pkg/types/ccipocr3/config.go

```diff
@@ -58,13 +58,19 @@ func (c CommitPluginConfig) Validate() error {
 
 type ExecutePluginConfig struct {
 	// DestChain is the ccip destination chain configured for the execute DON.
-	DestChain ChainSelector
+	DestChain ChainSelector `json:"destChain"`
 
 	// ObserverInfo is a map of oracle IDs to ObserverInfo.
-	ObserverInfo map[commontypes.OracleID]ObserverInfo
+	ObserverInfo map[commontypes.OracleID]ObserverInfo `json:"observerInfo"`
 
 	// MessageVisibilityInterval is the time interval for which the messages are visible by the plugin.
-	MessageVisibilityInterval time.Duration
+	MessageVisibilityInterval time.Duration `json:"messageVisibilityInterval"`
+
+	// FChain defines the FChain value for each chain. FChain is used while forming consensus based on the observations.
+	FChain map[ChainSelector]int `json:"fChain"`
+
+	// F is the ocr3 big F value for the execute plugin.
+	F int `json:"f"`
```

Comment:
Does this need to be in here? This is passed into the `NewReportingPlugin` call directly. FChain isn't, however.

Reply:
Oh nice. I'll use the reporting cfg instead.